spec/features/recode_url_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe RecodeUrl do
+
+    let(:app) { RecodeUrl.new(Rails.application) }
+    subject { described_class.new(app) }
+
+    let(:request) { Rack::MockRequest.new(subject) }
+  
+    context "when called with a GET request" do
+        before(:each) do
+            request.get("/accounts/api/user", 'CONTENT_TYPE' => 'text/plain')
+        end
+        it "changes PATH_INFO" do
+            expect(app["PATH_INFO"]).to eq('text/plain')
+        end
+    end
+end
```

Comment:
Yay!  glad `Rack::MockRequest` worked

Reply:
@KarinaMendez2 please use two spaces for indentation and make sure to use spaces instead of tabs.